### DISCUSSION TITLE
formerly perfectly fine downloads on putlocker does not work anymore

### DISCUSSION THREAD

**Discussion by Ratpack22** [aclap-dev/video-downloadhelper#1531] [Help and Q&A]
### Your question

Since a couple of weeks, I cannot download any videos anymore on my Windows PC either with my Google Chrome nor my Firefox browser Which both have the download helper add-ones installed. In both cases the videos are shown normally and  streaming normally. The download helper recognises the videos and shows me the download button. Yet when I want to download them after a couple of seconds, I get an error message that the download is not possible.
I use the latest versions of downloadHelper (9.1.0.44),  Firefox (133.0 on 64 bit) and Google Chrome ( 131.0.6778.109 on 64 bit).

Would be great if somebody could tell me how to repair this !

### Addon & CoApp version and details

I use the latest versions of downloadHelper (9.1.0.44),  Firefox (133.0 on 64 bit) and Google Chrome ( 131.0.6778.109 on 64 bit).

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You would do well to provide at least a couple of examples.  Saying every video is a problem for you isn't any help.  I'm able to download videos with VDH without any problem.  If somebody else is to investigate your problem, you have to supply some URLs of specific videos.  NOT names of web sites.  URLs of pages where videos reside.

It also occurs to me that others have reported similar total failures when their antivirus or firewall or both were interfering.  You should search this forum for discussions of VDH & antivirus.  There's plenty of them.  You'll find useful advice there.
